static/js/subs.js

```diff
@@ -1117,8 +1117,23 @@ $(function () {
         sub_arrow.removeClass('icon-vector-chevron-up');
         sub_arrow.addClass('icon-vector-chevron-down');
     });
+
+    //Set the width of subscriptions status box with respect to size of
+    //middle column.
+    function set_subscription_status_width() {
+        var column_middle_width = $('.column-middle').width();
+        $('#subscriptions-status').width(0.8*column_middle_width);
+    }
+
+    $(window).resize(function () {
+        set_subscription_status_width();
+    });
+
+    set_subscription_status_width();
```

Comment:
Found it, `app-main` has max-width of 1400px, changed it with respect to it.Made it so simple :)